modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/NodeLeftException.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.sql.engine;
+
+import static org.apache.ignite.lang.ErrorGroups.Sql.RUNTIME_EXECUTION_ERR;
+
+import org.apache.ignite.lang.IgniteException;
+
+/**
+ * The exception is thrown when SQL engine can not process an operation because a node has a left cluster.
+ */
+public class NodeLeftException extends IgniteException {
```

Comment:
Can we move this class to api module and use error code from Common group?

Reply:
Sure we can but I think do we really need NodeLeftException? Maybe we can use `IgniteException` with `NODE_LEFT_ERR` code.

Reply:
Moved NODE_LEFT to common error group.